tests/kafkatest/services/templates/kafka.properties

```diff
@@ -20,6 +20,8 @@ port=9092
 #host.name=localhost
 advertised.host.name={{ node.account.hostname }}
 #advertised.port=<port accessible by clients>
+listeners=PLAINTEXT://:9092,SSL://:9093
```

Comment:
@ijuma Are these suggested tests captured in JIRAs somewhere? I haven't tracked the SSL issues closely, but I think getting an initial patch for existing tests checked in is a good start, and then we (hopefully easily) add these other tests in a separate patch.